PSFzf.Base.ps1

```diff
@@ -836,6 +836,39 @@ function Invoke-FzfPsReadlineHandlerHistoryArgs {
 	}
 }
 
+function Invoke-FzfHandlerHistoryHost {
+	$Commands = @('icm','Invoke-Command','Enter-PSSession','etsn')
```

Comment:
That's a bit limiting, and seems case-sensitive? Should probably be using `(Get-Command $Name -Type Alias).ResolvedCommandname` (plus error checking) to get the command from an alias so that it also works for custom aliases, not just icm and etsn.

Reply:
Yes ! Maybe something like that :
```
	$Cmdlets = @('Invoke-Command','Enter-PSSession')
	$Alias = (get-alias |? { $Cmdlets -contains $_.ResolvedCommand }).Name
	$Commands = $Cmdlets + $Alias

```
And as in Powershell 5, Get-Service, Get-Process, Get-WmiObject used to have a "-ComputerName" parameters, this entries should be searched in history